spring-xd-related-work.tex

```diff
@@ -49,15 +49,65 @@ \subsection{Spring XD and Storm}
 \item Building upon the functional stream processing model, users have the option
 to choose from Reactor\cite{reactor}, Spark Streaming or RxJava APIs to build
 complex data centric applications.
+\item As an alternative to linear streaming data pipelines, Spring XD provides 
+support for directed ayclic graphs (DAG)\cite{dag} through named channels to 
```

Comment:
Would you agree the use of named channels could help building non-linear streams in Spring XD? I was trying to articulate just that, but of course I'm open to rephrasing it.